CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing Guidelines
+
+First of all, thanks for taking the time to contribute!
+
+Before starting to work on an issue, please go through the existing issues and pull requests to make sure no one is working on the same issue at the time.
+
+Once you select a task to work on, please mention that in the issue to let others to know you are working on that. If it's a new issue, please create an issue and do as same.
+
+## Setting up the development environment
+
+If you are not a member of the [@99xt](https://github.com/99xt) github orginzation, you should fork this repository, make changes in your own fork, and then submit a pull-request. 
```

Comment:
@dinukadesilva can you update both places? Good suggestion on creating a org wide template. For Issue template/ PR template and Code of Conduct this would work. We had few edge cases where contribution guidelines were changed. I will look into creating general templates. However as per my knowledge, each repo should contain Issue Templates and PR template files right?